### ISSUE TITLE
examples folder

### ISSUE THREAD

**Issue by bursteways** [saiichihashimoto/sanity-typed-schema-builder#199]
It would be great to see a few examples of how this fits into the Sanity schema config. It's probably trivial, but serves as nice resource for newcomers.

**Comment by saiichihashimoto** [comment]
I agree. I've tried working those into the README but it's all a bit verbose. I'm not the best at documentation, so I feel there's a docs rework that needs to happen here in general.
